mirrord-schema.json

```diff
@@ -756,6 +756,14 @@
       "description": "mirrord Experimental features. This shouldn't be used unless someone from MetalBear/mirrord tells you to.",
       "type": "object",
       "properties": {
+        "disable_reuseaddr": {
+          "title": "_experimental_ disable_reuseaddr {#experimental-disable_reuseaddr}",
+          "description": "Disables the `SO_REUSEADDR` socket option on sockets that mirrord steals/mirrors. on macOS the application can use the same address many times but then we don't steal it correctly. This probably should be on by default but we want to gradual roll it out. https://github.com/metalbear-co/mirrord/issues/2819 This option applies only on macOS.",
```

Comment:
```suggestion
          "description": "Disables the `SO_REUSEADDR` socket option on sockets that mirrord steals/mirrors. On macOS the application can use the same address many times but then we don't steal it correctly. This probably should be on by default but we want to gradually roll it out. https://github.com/metalbear-co/mirrord/issues/2819 This option applies only on macOS.",
```

Reply:
Thanks for the suggestion! P.S, next time send the review on the comments where it's being generated from :D